spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -71,6 +71,8 @@ public static void startMetastoreAndSpark() {
             .config(SQLConf.PARTITION_OVERWRITE_MODE().key(), "dynamic")
             .config("spark.hadoop." + METASTOREURIS.varname, hiveConf.get(METASTOREURIS.varname))
             .config("spark.sql.legacy.respectNullabilityInTextDatasetConversion", "true")
+            // Needed for Delta Lake tests
+            .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
```

Comment:
can we have another test base for this to avoid loading Delta extension for all Spark tests?

Reply:
I created a separate testbase called `SparkDeltaLakeMigrationTestBase` for the delta lake related task